.gitignore

```diff
@@ -1,3 +1,4 @@
+.github
```

Comment:
Is there a particular reason for excluding this folder?
This seems to me that it will cause issues if we want to add more actions in this folder in the future.

Reply:
Ooops, no. I'm not even sure why I added this. It doesn't really make sense. Will update the PR.